src/evaluate/evaluator/question_answering.py

```diff
@@ -0,0 +1,191 @@
+            metric (`str` or `EvaluationModule`, defaults to `None`):"
+                Specifies the metric we use in evaluator. If it is of type `str`, we treat it as the metric name, and
+                load it. Otherwise we assume it represents a pre-loaded metric.
+            tokenizer: (`str` or `PreTrainedTokenizer`, *optional*, defaults to `None`):
+                Argument can be used to overwrite a default tokenizer if `model_or_pipeline` represents a model for
+                which we build a pipeline. If `model_or_pipeline` is `None` or a pre-initialized pipeline, we ignore
+                this argument.
+            strategy: (`Literal["simple", "bootstrap"]`, defaults to "simple"):
+                specifies the evaluation strategy. Possible values are:
+                - `"simple"` - we evaluate the metric and return the scores.
+                - `"bootstrap"` - on top of computing the metric scores, we calculate the confidence interval for each
+                of the returned metric keys, using `scipy`'s `bootstrap` method
+                https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.bootstrap.html.
+            confidence_level (`float`, defaults to `0.95`):
+                The `confidence_level` value passed to `bootstrap` if `"bootstrap"` strategy is chosen.
+            n_resamples (`int`, defaults to `9999`):
+                The `n_resamples` value passed to `bootstrap` if `"bootstrap"` strategy is chosen.
+            random_state (`int`, *optional*, defaults to `None`):
+                The `random_state` value passed to `bootstrap` if `"bootstrap"` strategy is chosen. Useful for
+                debugging.
+            question_column (`str`, defaults to `"question"`):
+                the name of the column containing the question in the dataset specified by `data`.
+            context_column (`str`, defaults to `"context"`):
+                the name of the column containing the context in the dataset specified by `data`.
+            id_column (`str`, defaults to `"id"`):
+                the name of the column cointaing the identification field of the question and answer pair in the
+                dataset specified by `data`.
+            label_column (`str`, defaults to `"label"`):
+                the name of the column containing the answers in the dataset specified by `data`.
+        Return:
+            A `Dict`. The keys represent metric keys calculated for the `metric` spefied in function arguments. For the
+            `"simple"` strategy, the value is the metric score. For the `"bootstrap"` strategy, the value is a `Dict`
+            containing the score, the confidence interval and the standard error calculated for each metric key.
+        Examples:
+        ```python
+        >>> from evaluate import evaluator
+        >>> from datasets import Dataset, load_dataset
+        >>> e = evaluator("question-answering")
```

Comment:
we are setting the precedent here what people will use as default variable name for the evaluator. a single letter is not great. maybe at least `eval`? other ideas?

```suggestion
        >>> eval = evaluator("question-answering")
```